example_configs/flink-1-2.yml

```diff
@@ -0,0 +1,121 @@
+rules:
+  ## Job Manager ##
+  - pattern: org\.apache\.flink\.jobmanager\.([a-zA-Z\-\_]+)\.([a-zA-Z\-\_]+)\.([a-zA-Z\-\_]+)\.([a-zA-Z\-\_]+)\.([a-zA-Z\-\_]+)<host=(.*)><>(Count|Value|Rate)
+    name: flink_jobmanager_$1_$2_$3_$4_$5
```

Comment:
The jmx exporter will put in underscores for invalid characters, you can go just one capture group for $1-$5